### ISSUE TITLE
Interpolation closing brace is not tokenized 

### ISSUE THREAD

**Issue by mouhannad-sh** [sasstools/scss-tokenizer#20]
Hi I've tried the following example:

```javascript
tokenize(' #{$var +20} 10')
```

the output is 
```javascript
[ [ 'space', ' ' ],
  [ 'startInterpolant', '#{', 1, 3 ],
  [ '$', '$', 1, 4 ],
  [ 'ident', 'var', 1, 5, 1, 7 ],
  [ 'space', ' ' ],
  [ '+', '+', 1, 9 ],
  [ 'number', '20', 1, 10, 1, 11 ], // closing brace should be after this
  [ 'space', ' ' ],
  [ 'number', '10', 1, 14, 1, 15 ] ]
```

as you can see there's a `startInterpolant` token but the output doesn't indicate anything about the the closing brace.

Am I missing anything ? is there a different way to handle this syntax `#{ }` ? 

Thanks

**Comment by xzyfer** [comment]
Looks like you found a bug. I'll look into it this week

**Comment by xzyfer** [comment]
Fixed in 0.4.0
